packages/atomic/src/components/search/atomic-relevance-inspector/atomic-relevance-inspector.tsx

```diff
@@ -1,57 +1,59 @@
-import {
-  RelevanceInspector,
-  RelevanceInspectorState,
-  Unsubscribe,
-  buildRelevanceInspector,
-} from '@coveo/headless';
-import {Component, h, State, Prop} from '@stencil/core';
+import {Component, h, Prop, Event, EventEmitter} from '@stencil/core';
+import {Button} from '../../common/button';
 import {Bindings} from '../atomic-search-interface/atomic-search-interface';
 
 /**
  * The `atomic-relevance-inspector` component is used internally to offer insight on search page relevance, as well as information to help troubleshoot issues during development.
  */
 @Component({
   tag: 'atomic-relevance-inspector',
+  styleUrl: 'atomic-relevance-inspector.pcss',
   shadow: true,
 })
 export class AtomicRelevanceInspector {
-  public relevanceInspector!: RelevanceInspector;
-  private unsubscribe: Unsubscribe = () => {};
-
-  @State() relevanceInspectorState!: RelevanceInspectorState;
-
   /**
    * The Atomic interface bindings, namely the headless search engine and i18n instances.
    */
   @Prop() bindings!: Bindings;
 
-  constructor() {
-    this.relevanceInspector = buildRelevanceInspector(this.bindings.engine, {
-      initialState: {
-        // TODO: add enable/disable mechanism
-        enabled: false,
-      },
-    });
-    this.unsubscribe = this.relevanceInspector.subscribe(
-      () => (this.relevanceInspectorState = this.relevanceInspector.state)
-    );
-  }
+  @Prop({reflect: true}) open = false;
 
-  public disconnectedCallback() {
-    this.unsubscribe();
-  }
+  @Event({eventName: 'atomic/relevanceInspector/close'})
+  closeRelevanceInspector: EventEmitter<null> | undefined;
 
   public render() {
-    if (!this.relevanceInspectorState.isEnabled) {
-      return;
-    }
-
-    // TODO: Display data in a cleaner manner
+    const {platformUrl, organizationId} =
+      this.bindings.engine.state.configuration;
+    const {searchUid} = this.bindings.engine.state.search.response;
```

Comment:
If possible, I think we should try to get the searchUid associated with the particular search that returned the result which was clicked. I say this because we may see results from a previous query when somebody presses the "Show more" button.

Reply:
It's a fair point 🤔 

However, I do not see an easy way to achieve that except to add a new (cryptic) property on each results in Headless, that associates them with their originating search ID.

We do some shenanigans to make sure the search ID does not change on load more/facet search etc, so that analytics stay consistent.

Whereas here, we'd have to add a property that is only useful for this feature, and that "most people" should not bother with. Which I don't really like.

I'm tempted to leave it as is, and we'll see if it causes problems.

Also, in a somewhat near future, we'll add capabilities to modify the search parameters with the replay API.

Let's say that we show you the wrong page of results in the admin console, it should be possible to workaround that problem in the relevance inspector UI in the console.